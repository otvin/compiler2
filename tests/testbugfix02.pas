program testbugfix02(output);
{known bug from January 11, 2020; fixed January 25, 2020}

type myrange = -10..10;
var q:myrange;
begin
    q := -11;  {should be caught at compile-time, was then only caught at run-time}
end.
