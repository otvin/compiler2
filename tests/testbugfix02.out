Cannot assign value '-11' of type integer literal to type myrange in file:tests/testbugfix02.pas line:7     column:11 
